larray_editor/editor.py

```diff
@@ -657,13 +671,76 @@ def load_example(self):
                 self._open_file(filepath)
 
     def open_documentation(self):
-        QDesktopServices.openUrl(QUrl("http://larray.readthedocs.io/en/stable/"))
+        QDesktopServices.openUrl(QUrl(urls['doc_stable']))
 
     def open_tutorial(self):
-        QDesktopServices.openUrl(QUrl("http://larray.readthedocs.io/en/stable/tutorial.html"))
+        QDesktopServices.openUrl(QUrl(urls['doc_tutorial']))
 
     def open_api_documentation(self):
-        QDesktopServices.openUrl(QUrl("http://larray.readthedocs.io/en/stable/api.html"))
+        QDesktopServices.openUrl(QUrl(urls['doc_api']))
+
+    def report_issue(self):
+        if PY2:
+            from urllib import quote
+        else:
+            from urllib.parse import quote
+
+        versions = get_versions()
+        issue_template = """\
+## Description
+**What steps will reproduce the problem?**
+1. 
+2. 
+3.
+ 
+**What is the expected output? What do you see instead?**
+
+
+**Please provide any additional information below**
+
+
+## Version and main components
+* Python {python} on {system} {bitness:d}bits
+* Qt {qt}, {qt_api} {qt_api_ver}
+"""
+        for dep in dependencies:
+            issue_template += "* {dep} {{{dep}}}\n".format(dep=dep)
+        issue_template = issue_template.format(**versions)
+
+        url = QUrl(urls['new_issue'])
+        if PYQT5:
+            from qtpy.QtCore import QUrlQuery
+            query = QUrlQuery()
+            query.addQueryItem("body", quote(issue_template))
+            url.setQuery(query)
+        else:
+            url.addEncodedQueryItem("body", quote(issue_template))
+        QDesktopServices.openUrl(url)
+
+    def open_support(self):
+        QDesktopServices.openUrl(QUrl(urls['users_group']))
+
+    def open_releases_list(self):
+        QDesktopServices.openUrl(QUrl(urls['announce_group']))
+
+    def about(self):
+        """About Editor"""
+        kwargs = get_versions()
+        kwargs.update(urls)
+        message = """\
+<b>LArray Editor {editor}</b>: The Graphical User Interface for LArray. 
+<br>Licensed under the terms of the <a href="{GPL3}">GNU GENERAL PUBLIC LICENSE Version 3</a>.
+<p>Developed and maintained by the <a href="{fpb}">Federal Planning Bureau</a> (Belgium). 
+<p><b>Versions of underlying libraries</b> (please copy-paste this when reporting a bug):
```

Comment:
Given this is done automatically by the report issue link, the please copy-paste sentence is now mostly useless.

Reply:
I forgot to remove that, sorry.